### ISSUE TITLE
[Feature]Lock note no need enter password

### ISSUE THREAD

**Issue by wu4339** [federicoiosue/Omni-Notes#972]

**Comment by federicoiosue** [comment]
Hi there, maybe I got it wrong, you meant:

- there is no need to request for the password when opening a new locked note
- password should be requested when opening a new locked note but actually it is not

**Comment by wu4339** [comment]
> Hi there, maybe I got it wrong, you meant:
> 
> - there is no need to request for the password when opening a new locked note
> - password should be requested when opening a new locked note but actually it is not

I meant : lock a new note no need to request the password. because the note password all the same one.

**Comment by federicoiosue** [comment]
I agree that it may be a little ripetitive but that behavior ensure that the person that is creating a new locked note (I really should rename them as "protected") knows the password to unlock it later and is the "app owner".
